docs/guides/cli/20_proxy-and-tunnel.mdx

```diff
@@ -0,0 +1,376 @@
+---
+id: proxy-and-tunnel
+title: Ory Proxy and Ory Tunnel
+---
+
+The CLI contains two important tools for local development and self-hosted instances
+of the Ory Identity Service (Kratos): 
+
+- Ory Proxy
+- Ory Tunnel
+
+Before you start using these tools, export the SDK of your Ory Cloud project. Run this command: 
+
+```shell
+export ORY_SDK_URL=https://{your-project-slug-here}.projects.oryapis.com
+
+# For a quick way to test, use the playground project: 
+export ORY_SDK_URL=https://playground.projects.oryapis.com
+```
+:::tip
+
+To get your project's SDK URL, sign in at
+[console.ory.sh](https://console.ory.sh/), select **Connect** from the left
+navigation panel, and copy the URL from the **SDK Configuration** section.
+
+:::
+
+## Ory Proxy 
+
+Ory Proxy is a reverse proxy deployed in front of your application. This allows the 
+Ory endpoints to be mirrored on the same domain as the app you're running. 
+Additionally, the Proxy converts sessions into JSON Web Tokens (JWTs) and ensures that cookies 
+and URLs are properly configured.
```

Comment:
Yes, but as @gen1us2k said, we need to be a  bit more informative here than just the general "it helps you". 

See my comment above ^

The proxy rewrites the cookies to match the domain your application is currently on (either the proxy domain of localhost:4000 or example.com). So instead of the cookie being set to `project-slug.projects.oryapis.com` it is now correctly set to the domain you own. (with cname this isn't necessary - since their Ory project is served under the domain name they own, but for local development they will still need o use the proxy/tunnel)